ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -441,7 +438,11 @@ objects:
         wait "$!"
         install_exit_status=$?
         sleep 10m
-        oc get co/image-registry
+        # This is a workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1794714
+        for n in $(oc --insecure-skip-tls-verify get nodes|awk '{print $1}'|grep worker);do
```

Comment:
I don't like operating on the table output of `get nodes`.  Can you use `-o jsonpath=...` to have it spit out the field you want?

Reply:
sure didn't know the option, I will try it